src/Frame.h

```diff
@@ -36,6 +36,8 @@ class Frame {
     ~Frame();
     /// Set up empty frame for given # of atoms.
     Frame(int);
+    /// Set up Frame as a memoryview. mostly for pytraj.
+    Frame(int, double*);
```

Comment:
@swails 

what's usage of `..double*, double*)`? for V_ too?

If I don't want to allocate V_, just use NULL, right?